src/main/java/org/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatter.java

```diff
@@ -26,7 +26,7 @@ public class HtmlToLatexFormatter extends Formatter implements LayoutFormatter {
 
     @Override
     public String format(String text) {
-        String result = Objects.requireNonNull(text);
+        String result = Objects.requireNonNullElse(text, "");
```

Comment:
Although this solves the problem, I'm not a fan. I believe it is the client's responsibility to use valid arguments when calling the format method. If it does not, there may be a larger issue in the app that is causing this behavior. 
I believe you can apply the fix here and it will fix them all.

https://github.com/JabRef/jabref/blob/0f3b01696a610f4b0b319d6253c7f71ed35843c8/src/main/java/org/jabref/gui/fieldeditors/contextmenu/DefaultMenu.java#L58-L68.

Reply:
Thanks for the feedback :)
Thinking about it, would it be better to disable the menu items, if there's no text present?

Reply:
Nice idea! Preventing the exception is already an improvement and it should be enough to get the PR accepted but if you want to take it to the next level and disable menu items that's up to you.

Reply:
Great to hear, I went for that solution. The tests are excluded as I don't think they are necessary when the function isn't available to the user anymore.